server/src/api/account.py

```diff
@@ -47,9 +47,22 @@ def register(self):
     
     @cherrypy.expose
     def login(self):
-        """Will verify user identity by checking user and password on user database
+        """Will verify user identity by checking user and password on user table
         
         Arguments:
-            json -- check api doc for json format
+            username (string): Name used for login
+            password (string): Password used for login
         """
-        pass
+        JSON_object = json.loads(cherrypy.request.body.read().decode('utf-8'))
+        username = JSON_object.get("username")
+        password = JSON_object.get("password")
+
+        try:
+            isLoggedIn = self.database.get_password(username) == password
+        except Exception as e:
+            print(e)
+        finally:
+            response = {
+                'success': isLoggedIn
```

Comment:
what happens if get_password throws an exception? You're catching the exception but I don't think there's a default value for isLoggedIn when accessed from finally.